README.md

```diff
@@ -222,6 +222,39 @@ dip run bash -c pwd
 
 returned is `/app/sub-project-dir`.
 
+#### $DIP_CURRENT_USER
+
+Exposes the current user ID (UID). It is useful when you need to run a container with the same user as the host machine. For example:
+
+```yml
+# dip.yml (1)
+environment:
+  UID: ${DIP_CURRENT_USER}
+```
+
+```yml
+# docker-compose.yml (2)
+services:
+  app:
+    build:
+      context: ./
+      dockerfile: Dockerfile
+      target: development
+      args:
+        UID: ${UID:-1000}
```

Comment:
So, you want to create a container for a specific user. I thought it would be [like this](https://medium.com/redbubble/running-a-docker-container-as-a-non-root-user-7d2e00f8ee15). Doesn't that work for you? My concern is that I don't always build an image for a service. More often, I use official images with no changes.

```yaml
services:
  app:
    image: ruby
    user: ${CURRENT_UID}
```

Reply:
So for example, with that exact setup, this is the terminal that I get when I launch bash.

![CleanShot 2024-02-27 at 16 01 41](https://github.com/bibendi/dip/assets/880387/a8963757-b278-4fb3-a93f-bbbbc549303b)

In our setup with the custom dockerfile (which was based on Ruby on Whales!), we are creating the user during the development stage and adding sudo privileges to it, etc. for an easier development workflow

Reply:
Thanks! I just wanted to make sure it works both ways :)